build.sbt

```diff
@@ -16,59 +15,48 @@ lazy val `scala-2-to-scala-3-master` = (project in file("."))
     `exercise_009_opaque_type_aliases`,
     `exercise_010_multiversal_equality`
  )
-  .settings(scalaVersion := dottyVersion)
+  .settings(scalaVersion in ThisBuild := CommonSettings.dottyVersion)
```

Comment:
```suggestion
  .settings(scalaVersion in ThisBuild := Version.scalaVersion)
```

Reply:
I've updated the branch with your proposed approach. `dottyVersion` is in `Version` now.